.github/workflows/test-enos-scenario-ui.yml

```diff
@@ -0,0 +1,128 @@
+---
+name: Vault UI Tests
+
+on:
+  workflow_call:
+    inputs:
+      test_filter:
+        type: string
+        description: "A filter to limit the ui tests to. Will be appended to the ember test command as '-f=<filter>'"
+        required: false
+      storage_backend:
+        type: string
+        description: "The storage backend to use, either 'raft' or 'consul'"
+        default: raft
+  workflow_dispatch:
+    inputs:
+      test_filter:
+        type: string
+        description: "A filter to limit the ui tests to. Will be appended to the ember test command as '-f=<filter>'"
+        required: false
+      storage_backend:
+        description: "The storage backend to use, either 'raft' or 'consul'"
+        required: true
+        default: raft
+        type: choice
+        options:
+          - raft
+          - consul
+  push:
+    branches:
+      - mbaum/qt-304
```

Comment:
note to be removed before merge

Reply:
will do..., just added it recently to test the changes I made per the code review